libraries/AP_NMEA_Output/AP_NMEA_Output.cpp

```diff
@@ -163,7 +164,55 @@ void AP_NMEA_Output::update()
     char rmc_end[6];
     snprintf(rmc_end, sizeof(rmc_end), "*%02X\r\n", (unsigned) _nmea_checksum(rmc));
 
-    const uint32_t space_required = strlen(gga) + strlen(gga_end) + strlen(rmc) + strlen(rmc_end);
+
+    // get roll, pitch, yaw
+    const float roll_deg = wrap_180(degrees(ahrs.get_roll()));
+    const float pitch_deg = wrap_180(degrees(ahrs.get_pitch()));
+    const float yaw_deg = wrap_360(degrees(ahrs.get_yaw()));
+    const float heave_m = 0; // instantaneous heave in meters
+    const float roll_deg_accuracy = 0; // stddev of roll_deg;
+    const float pitch_deg_accuracy = 0; // stddev of pitch_deg;
+    const float heading_deg_accuracy = 0; // stddev of yaw_deg;
```

Comment:
Is this correct according to spec or should the field be empty or something?

Reply:
I don't know, your guess is as good as mine. Safer to put it in than not?